index.html

```diff
@@ -3210,7 +3220,7 @@ <h1>Artists</h1>
             <p id="statPoM5" alt="statPoM5" label="statPoM5" class="statPortion hepteracts">Cube upgrade 1x1:<span id="sPoM5" alt="sPoM5" label="sPoM5" class="statNumber">0</span></p>
             <p id="statPoM6" alt="statPoM6" label="statPoM6" class="statPortion hepteracts">Cube upgrade 2x1: <span id="sPoM6" alt="sPoM6" label="sPoM6" class="statNumber">0</span></p>
             <p id="statPoM7" alt="statPoM7" label="statPoM7" class="statPortion hepteracts">Cube upgrade 2x1: <span id="sPoM7" alt="sPoM7" label="sPoM7" class="statNumber">0</span></p>
-            <p id="statPoMT" alt="statPoMT" label="statPoMT" class="statPortion statTotal" style="color: orchid">FINAL COEFFICIENT: <span
+            <p id="statPoMT" alt="statPoMT" label="statPoMT" class="statPortion statTotal">FINAL COEFFICIENT: <span
```

Comment:
was the color removed here purposefully?

Reply:
This colors, cant be done as class (at least in past), because color get overwritten, so color was moved into CSS
Because I needed var()